pkg/controller/keyvault/kv_util.go

```diff
@@ -16,40 +15,60 @@ import (
 
 var nginxNamePrefix = "keyvault-nginx-"
 
-func buildSPC(obj client.Object, spc *secv1.SecretProviderClass, config *config.Config) (bool, error) {
-	var certURI, specSecretName string
+type EventType int
```

Comment:
where is this used?

Reply:
I'd rather just have the constant than hardcode the string in multiple places to prevent typos/invalid event types. Here's a summary of the usage though:
![image](https://github.com/user-attachments/assets/2be88447-bd4a-40f4-8c8d-ae57c3a53a19)

Reply:
this isn't in the k8s libraries anywhere?

Reply:
It wasn't in the events library but I spent some time digging through the code and found it - replacing now, thanks for pointing this out